### PR TITLE
CREATE test_file for test purposes

### PR DESCRIPTION
**Pull request by SergeiAP**
Colleagues, please review the code

### CODE REVIEW

**Comment by SergeiAP** [comment]
I see that the file is empty. Please, add something:-)

**Comment by Sergei-A-P** [comment]
Please, add something to the file

**Review by Sergei-A-P — CHANGES_REQUESTED** [review]
Please, add something to the file

**Comment by SergeiAP** [comment]
> Please, add something to the file

Ok

**Comment by Sergei-A-P** [comment]
#2

**Review by Sergei-A-P — COMMENTED** [review]
Comment

**Comment by Sergei-A-P** [comment]
#2

File: test_file [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Write something
+fffff
```
Comment (Sergei-A-P) [inline test_file:2]:
```suggestion
bla-lba
```

**Review by Sergei-A-P — APPROVED** [review]

**Review by Sergei-A-P — APPROVED** [review]

**Review by Sergei-A-P — APPROVED** [review]

**Pull request closed without merging**